src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
+// Working with data
+// Making array from textContant
 
-// write your code here
+const populationData = Array.from(document.querySelectorAll('span.population'));
+
+// Convert data to String
+const dataToString = populationData.map((span) => span.textContent);
+
+// Convert data to Number
+const dataToNums = dataToString.map((str) => +str.replaceAll(',', ''));
```

Comment:
Using `+str` is a concise way to convert a string to a number, but it might be less readable for some developers. Consider using `Number(str)` for clarity.